.travis.yml

```diff
@@ -5,4 +5,6 @@ matrix:
     - python: '3.6'
       install:
       - pip install flake8
-      script: flake8
+      script:
+      - flake8
+      - flake8 `find bin -type f | while read in ; do if file --mime-type "${in}" | grep -q python ; then echo "${in}" ; fi ; done;`
```

Comment:
This is more convoluted than I think we really need. The while loop and if statements make it a bit hard to follow. Something like this seems to be cleaner and shorter and gives the same answer:
```
find bin -type f -exec file --mime-type {} \; | grep python | cut -f1 -d:
```
oh, and `$()` syntax is preferred over backticks these days I think.

Reply:
Thanks for the ninja trick -- updated.